lib/BeaconDriver.js

```diff
@@ -32,44 +33,67 @@ class BeaconDriver extends Homey.Driver {
 
                 // set available, it is possibly marked unavailable in v1.2.7
                 // @deprecated
-                device.setAvailable();
```

Comment:
The `setAvailable` is normally used if the device isn't properly connected. I don't think we should use this to determine if an beacon is in range because i think a lot of users think it is broken.

Reply:
In my Homey  I have a Kodi device (a Windows PC with a TV tuner; it is my "media center"). When it is turned off (the most of the time) Homey shows it with an exclamation point. For me this is normal and not alarming...
mine was just a suggestion from my everyday user experience (I wrote "experimental").